### PR TITLE
Nuzhy/Footer block and component

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
image

### CODE REVIEW

File: libs/blocks/src/lib/footer/footer.social.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Text } from '@deriv/quill-design';
+import { SocialIcons } from '.';
+
+export const SocialMediaSection = ({
+  socialButtons,
+}: {
+  socialButtons: SocialIcons[];
+}) => {
+  return (
+    <div className="flex flex-row gap-gap-lg max-lg:justify-center">
```
Comment (prince-deriv) [inline libs/blocks/src/lib/footer/footer.social.tsx:10]:
```suggestion
    <div className="flex gap-gap-lg max-lg:justify-center">
```

`flex` direction is `row` by default

**Review by meenakshi-deriv — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]
please update the branch with latest main, thanks 🙏

File: apps/deriv-hk/pages/help-centre.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 import { HelpCenterTemplate } from '@deriv-com/templates';
 const HelpCenterPage = () => {
   return (
     <>
       <Navigation.Main />
       <HelpCenterTemplate />
+      <Footer.Main />
     </>
   );
 };
```
Comment (prince-deriv) [inline apps/deriv-hk/pages/help-centre.tsx:11]: @mohsen-deriv  are we planning to have another layer like `layout` that will have the `navigation` and `footer` so we don't need to add them on each pages?

Comment (mohsen-deriv) [reply apps/deriv-hk/pages/help-centre.tsx:11]: for now let's keep like this and when we're done identifying all the navigation variants, we can create the layout.

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**